backend/src/js/addToCartFromThumbs.js

```diff
@@ -54,38 +47,28 @@ function addProductToCart(e) {
             price: normalPrice,
             quantity,
             total,
-
         };
 
-
-
         orders.push(product);
 
         // Writing added products to local storage
         cart.setItem("cart", JSON.stringify(orders));
         Notiflix.Notify.success('Product successfully added to cart');
 
     }
-
-
 }
 
 // Listener for loading more products
-async function loadMore(e) {
+async function loadMore() {
     const page = Number(window.location.search.slice(6)[0]);
```

Comment:
Wrong approach that will lead to lots of bugs in the future. Change the way retrieving the page